plugins/core/webview/src/q-ui/components/ssoLoginForm.vue

```diff
@@ -39,6 +39,10 @@
                 name="regions"
                 v-model="selectedRegion"
                 @change="handleRegionInput"
+                size="1"
+                onfocus='this.size=10;'
+                onblur='this.size=1;'
```

Comment:
do we need to explicitly set onBlur?

Reply:
You're right we don't need to explicity set the Onblur if we set it onChange.